### DISCUSSION TITLE
Alpha 160

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4843] [Release Notes]
- Added a "duplicate tab" hotkey (by @Me1onRind) [#4775](https://github.com/Eugeny/tabby/pull/4775)
- Added more timeout options for the Vault password (by Aaron Davison) [#4779](https://github.com/Eugeny/tabby/pull/4779)
- Made environment var overriding case insensitive on Windows
- Allowed variable substitution for env vars (e.g. `PATH = $PATH:/opt/...`) 
- SSH: Fixed X11 port detection for UNIX sockets
- SSH: Allowed overriding X11 display (Settings -> SSH)

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.160'>Alpha 160</a>.</em>

**Comment by FlameMida** [comment]
Unsupported algorithm: aes128-gcm when using ssh

**Reply by zack709** [reply]
Unsupported algorithm: aes128-gcm + 1

**Reply by kphk2** [reply]
Same problem

**Reply by yydog** [reply]
Delete the old ssh config
Re-add the config

**Comment by eladkapusta** [comment]
For me, overriding the default SSH socket defined by Tabby doesn't work for defined sessions. [Running latest ubuntu]
I thought maybe exiting and opening it again will use the manual settings but it appears it just doesn't change
For now, is there a manual way of overriding it, except launching an ssh session from a local terminal inside tabby?

**Reply by Eugeny** [reply]
Do you mean the X11 or the ssh-agent socket?

**Reply by eladkapusta** [reply]
X11, the default value for me is /tmp/.X11-unix/X0 for example, while in practice it should be /tmp/.X11-unix/X1.
Changing it in the SSH settings in Tabby and then reopening tabby doesn't seem to change anything, although it does appear to be saved in the SSH settings, it is not used in practice in the sessions.

**Reply by Eugeny** [reply]
Fixed in the nightly!
